src/algebra/field.lean

```diff
@@ -23,14 +18,14 @@ instance division_ring.to_domain [s : division_ring α] : domain α :=
 
 @[simp] theorem inv_one [division_ring α] : (1⁻¹ : α) = 1 := by rw [inv_eq_one_div, one_div_one]
 
-@[simp] theorem inv_inv' [discrete_field α] (x : α) : x⁻¹⁻¹ = x :=
-if h : x = 0
-then by rw [h, inv_zero, inv_zero]
-else division_ring.inv_inv h
+attribute [simp] inv_inv'
 
-lemma inv_involutive' [discrete_field α] : function.involutive (has_inv.inv : α → α) :=
+lemma inv_inv'' [division_ring α] (x : α) : x⁻¹⁻¹ = x :=
 inv_inv'
```

Comment:
What's the difference between `inv_inv'` and `inv_inv''`? Why do we need both?

Reply:
As far as I can see, the only difference is that `inv_inv'` takes `x` as an implicit argument. I think, this should be fixed in the stdlib.